kubernetes/helm_charts/core/adminutils/values.j2

```diff
@@ -59,3 +60,10 @@ adminutilenv:
   ACCESS_TOKEN_VALIDITY: '"{{ adminutil_access_token_validity }}"'
   REFRESH_TOKEN_OFFLINE_VALIDITY: '"{{ adminutil_refresh_token_offline_validity }}"'
   REFRESH_TOKEN_LOG_OLDER_THAN: '"{{  adminutil_refresh_token_log_older_than }}"'
+
+autoscaling:
+  enabled: {{ adminutils_autoscaling_enabled | default('false') }}
+  minReplicas: {{ adminutils_autoscaling_minReplicas|default(1) }}
+  maxReplicas: {{ adminutils_autoscaling_maxReplicas|default(4) }}
+  targetCPUUtilizationPercentage: {{ adminutils_autoscaling_targetCPUUtilizationPercentage|default(80) }}
+  #targetMemoryUtilizationPercentage: {{ adminutils_autoscaling_targetMemoryUtilizationPercentage|default(80) }}
```

Comment:
uncomment this line and change the default to `''` so that if we want to override, we can.